src/Helpers/SearchableFinder.php

```diff
@@ -105,4 +120,25 @@ private function getProjectClasses(): array
 
         return self::$declaredClasses;
     }
+
+    /**
+     * Using the laravel project's composer.json retrieve the PSR-4 autoload to determine
+     * the paths to search and namespaces to check against.
+     *
+     * @return void
+     */
+    private function inferProjectSourcePaths(): void
```

Comment:
Return `$sources` and a `$namespaces` from this method instead of setting it into attributes of the class.